test/ZNSCurvePricer.test.ts

```diff
@@ -50,14 +51,22 @@ describe("ZNSCurvePricer", () => {
       randomAcc,
     ] = await hre.ethers.getSigners();
 
+    const priceConfig : ICurvePriceConfig  = DEFAULT_PRICE_CONFIG;
+    const zeroVaultAddress : string = deployer.address;
+    const isTenderlyRun  = false;
+
     zns = await deployZNS({
       deployer,
       governorAddresses: [deployer.address],
       adminAddresses: [admin.address],
+      priceConfig,
+      zeroVaultAddress,
+      isTenderlyRun,
     });
 
-    await zns.meowToken.connect(user).approve(await zns.treasury.getAddress(), ethers.MaxUint256);
-    await zns.meowToken.mint(user.address, DEFAULT_PRICE_CONFIG.maxPrice);
+    const userBalanceInitial = await zns.zToken.balanceOf(admin.address) / 2n;
```

Comment:
Why `/ 2n` ? This calls back to other comments I've given you in the past, please always comment things to explain what its doing or why, especially for seemingly arbitrary numbers